libs/components/lookup/src/lib/modules/search/search.component.scss

```diff
@@ -123,37 +127,77 @@ sky-search {
   .sky-search-dismiss-absolute {
     align-items: center;
     position: absolute;
-    background-color: lookupVars.$sky-search-background-color;
+    background-color: var(
+      --sky-override-search-mobile-background-color,
+      var(--sky-color-background-action-secondary-base)
+    );
     top: 0;
     left: 0;
     bottom: 0;
     right: 0;
-    padding: $sky-padding-half;
+    padding: var(
+      --sky-override-search-container-padding,
+      var(--sky-space-inset-balanced-xs)
+    );
   }
 }
 
 .sky-theme-modern {
   sky-search {
     .sky-input-box {
-      --sky-background-color-input-box-group: transparent;
+      --sky-background-color-input-box-group: var(
+        --sky-override-search-background-color,
+        var(--sky-color-background-action-secondary-base)
+      );
     }
 
     .sky-input-box .sky-input-group-btn .sky-btn.sky-search-btn {
-      width: 40px;
+      height: var(
+        --sky-override-search-button-height-width,
+        calc(
```

Comment:
Is there a reason that these calculations are slightly different than what is in the spec?

```
height: calc(var(--sky-line_height-body-m) + calc(var(--sky-spacing-inset-balanced-s) * 2));
```

Reply:
yes, line height needs to be multiplied by font size to get an actual height value there, and the `body-m` was incorrect, input boxes set a different line height